apps/ia/demo/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+patchesStrategicMerge:
+  - ../../ia-hearings-api/demo.yaml
+  - ../../identity/aat.yaml
```

Comment:
Is demo going to use aat identity? you should have a demo identity

Reply:
Thanks Thomas, added this under the latest commit